src/renderer/mod.rs

```diff
@@ -44,15 +47,16 @@ impl<'a> Renderer<'a> {
 
     /// Combines the context with the Template to generate the end result
     pub fn render(&self) -> Result<String> {
-        let output;
-
-        {
-            let mut processor =
-                Processor::new(self.template, self.tera, &self.context, self.should_escape);
+        let mut output = Vec::with_capacity(10_000);
```

Comment:
any reason 10k was chosen?

Reply:
The only reason is that previous code allocate a String buffer of 10.000 chars: https://github.com/Keats/tera/pull/601/files#diff-e67fd4dbfd06da05000d6b37dcecbf48e765d6e30eb90d021fa01fae7b094887L1027

Reply:
oh I forgot about that